dispatches/models/nuclear_case/hydrogen_tank.py

```diff
@@ -251,70 +233,88 @@ def volume_cons(b, t):
             within=Reals,
             initialize=1.0,
             doc="Energy accumulation",
-            units=power_units)
+            units=units("power"))
 
         self.material_holdup = Var(
             self.flowsheet().config.time,
             pc_set,
             within=Reals,
             initialize=1.0,
             doc="Material holdup in tank",
-            units=flow_units*time_units)
+            units=flow_units*units("time"))
```

Comment:
Why not just `units("amount")` or `units("mass")` as appropriate? The property package should tell you the basis for flow using `MaterialFlowBasis`.

This could also be important when doing the energy balance (to make sure units are consistent).